### DISCUSSION TITLE
Need clarification about license and attribution

### DISCUSSION THREAD

**Discussion by subham8907** [uBlockOrigin/uBOL-home#260] [Q&A]
Hey Gorhill or any one can you clarify that is this file https://github.com/uBlockOrigin/uBOL-home/blob/main/chromium/rulesets/removeparam/default.json is licensed under GPL 3.0 only or Later version also please if i use this file in my project can clarify anyone copyright holder as well with year

**Comment by subham8907** [comment]
Please clarify i want to use your default.json in your project Also i want to convey thank you for contributing to FOSS community

**Comment by MasterKia** [comment]
That file is the result of converting uAssets filterlists (GPLv3+) into DNR rulesets so it carries the same _GPL 3.0 or later_ license.

**Reply by ghost** [reply]
As I understand it's from "default" ruleset, so it also includes other lists, but these other lists are unlikely to include "removeparam" filters.

https://github.com/gorhill/uBlock/blob/143cc0280ad93071bf3223f0618763d994661cbe/platform/mv3/make-rulesets.js#L1195-L1203

Also, it's a result of compiling these lists, and it's distributed uder GPLv3+, so... I'm not a lawyer ;)

**Comment by subham8907** [comment]
thank you for clarification
